tokens/manage/proxy-accounts.md

```diff
@@ -107,27 +124,39 @@ As previously mentioned, you can also create a proxy from the **Accounts** page.
 1. Select the 3 vertical dots next to the primary account
 2. Select **Add proxy**
 
-![Select the Add proxy menu item from the Accounts page of Polkadot.js Apps.](/images/tokens/manage/proxy-accounts/proxies-2.webp)
+![Select the Add proxy menu item from the Accounts page of Polkadot.js Apps.](/images/tokens/manage/proxy-accounts/proxies-3.webp)
 
 !!! note
     If the account already has a proxy, **Manage proxies** will be displayed as an option instead of **Add proxy**.
 
 A pop-up will appear and you will be able to enter in the required information, such as the proxied/primary account, the proxy account, and type of proxy in order to create a proxy account. First click **Add Proxy**.
 
-![Add a proxy account from the Accounts page of Polkadot.js Apps](/images/tokens/manage/proxy-accounts/proxies-3.webp)
+![Add a proxy account from the Accounts page of Polkadot.js Apps](/images/tokens/manage/proxy-accounts/proxies-4.webp)
 
 Then take the following steps:
 
 1. Select the account you would like to set as a proxy
 2. Select the proxy type
 3. Click **Submit** and sign the transaction
 
-![Add the details of the proxy account, including the proxy account and type.](/images/tokens/manage/proxy-accounts/proxies-4.webp)
+![Add the details of the proxy account, including the proxy account and type.](/images/tokens/manage/proxy-accounts/proxies-5.webp)
 
 In the next section, you will learn how to verify that your proxy account was set up successfully.
 
 ## Verifying your Proxy Account {: #verifying-your-proxy-account }
 
+### Via Apps.Moonbeam.Network {: #verify-via-apps-moonbeam-network }
+
+When connected to [apps.moonbeam.network](https://apps.moonbeam.network/moonbeam/proxy){target=\_blank} with your primary account, you can see a list of accounts with proxy control over your connected primary account in the **Your Proxies** section.
```

Comment:
```suggestion
When connected to [the Moonbeam dApp](https://apps.moonbeam.network/moonbeam/proxy){target=\_blank} with your primary account, you can see a list of accounts with proxy control over your connected primary account in the **Your Proxies** section.
```

Using the URL as the link text will result in some of my reports returning issues with descriptive link text. It's an accessibility thing